services/distribution/src/main/resources/application.yaml

```diff
@@ -86,7 +85,7 @@ services:
     app-features:
       - label: reserved
         value: ${reserved:1}
-    supported-countries: ${SUPPORTED_COUNTRIES:DE}
+    supported-countries: ${SUPPORTED_COUNTRIES:DE,FR}
```

Comment:
Should there be FR as default for this environment variable or should it just be a list with DE?

Reply:
as long as we don't have 100% confirmation, I would like to have only 'DE' as default
when the piloting phase starts we might have further countries to add